src/test/packages/10-cosign-artifacts/zarf.yaml

```diff
@@ -0,0 +1,9 @@
+kind: ZarfPackageConfig
+metadata:
+  name: cosign-artifacts
+  description: Test package for cosign image artifact pulls
+
+components:
+  - name: test
+    images:
+      - ghcr.io/defenseunicorns/zarf/agent:sha256-90863c246da361499e8f59dfae728c34b50ee2057e61e06dacdcfad983425c32.sig
```

Comment:
Can we integrate this into a podinfo example instead (they also sign their image): https://github.com/stefanprodan/podinfo/pkgs/container/podinfo (that may keep things simpler and more discoverable if it is just in another example and it has a comment explaining why it is there)

Reply:
Moved to the podinfo helm-charts example and removed the separate test for this.